conf/test_full.config

```diff
@@ -22,7 +22,7 @@ params {
     map             = "/groups/dog/llenezet/test-datasets/data/genetic_maps.b38/chr21.b38.gmap.gz"
     genome          = "GRCh38"
     fasta           = "/groups/dog/llenezet/script/phaseimpute/data/genome.fa"
```

Comment:
I feel like these path are a bit too specific to your system, but that can be fix later on, just don't forget about it, otherwise the megatests will fail badly

Reply:
For sure.
I'm currently working on the creation of such dataset